### ISSUE TITLE
RAdamSchedulefree broken?

### ISSUE THREAD

**Issue by PaleKemono** [67372a/LoRA_Easy_Training_Scripts#43]
**Describe the bug**
Using RAdamSchedulefree optimizer results in an instant NaN loss, thus resulting in unsuccessful training,

**To Reproduce**
Select the RAdamSchedulefree optimizer, start training. Loss becomes NaN after the second training step

**Additional context**
There is this warning in console " WARNING  Encountered an error while trying to determine default orthograd from train_network.py:1065                             optimizer init signature. module 'torch.optim' has no attribute
                             'RAdamScheduleFree' "

<img width="969" height="382" alt="Image" src="https://github.com/user-attachments/assets/52169621-8467-4b59-b683-e1ddf00d056c" />

im using these settings, latest commit

[Schedfree.txt](https://github.com/user-attachments/files/21565057/Schedfree.txt)
